ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -587,9 +591,7 @@ export default defineComponent({
       alertMsg: computed((): string => {
```

Comment:
The prop name should be updated, since there is no more message but only the date.

Reply:
Modified

Reply:
Don't see it modified.